tests/e2e/test_skills_e2e.py

```
@@ -0,0 +1,320 @@
+# ---------------------------------------------------------------------------
+# Tier 1 – SkillRegistry
+# ---------------------------------------------------------------------------
+
+
+class TestSkillRegistry:
+    def _make_registry(self, name: str, body: str, requirements: tuple = ()) -> SkillRegistry:
+        entry = SkillEntry(
+            name=name,
+            description=f"{name} description",
+            body=body,
+            source=f"/tmp/{name}/SKILL.md",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)